11-payment-encoding.md

```diff
@@ -235,6 +239,9 @@ by which the description is served is as-yet unspecified and will
 probably be transport dependent. The `h` format could change in the future,
 by changing the length, so readers ignore it if it's not 256 bits.
 
+The `m` field allows metadata to be attached to the payment. This supports
```

Comment:
Even if we don't set a hard rule in the spec on the maximum size of this field, implementations will likely set a threshold to ensure there is enough space in the onion for other fields and a long enough route, so we should at least mention it here.

Reply:
Added